internal/pkg/credential/process_test.go

```diff
@@ -0,0 +1,31 @@
+package credential
+
+import (
+	"errors"
+	"strings"
+	"testing"
+)
+
+func TestGetToken_SingleCommand(t *testing.T) {
+	_, err := GetToken([]string{"test"})
+	if !strings.Contains(err.Error(), "exit status 1") {
+		t.Errorf("GetToken got error = %v, want exit status 1", err)
+	}
+}
```

Comment:
Just realized the single command scenario is considered. Not understanding why it is expected to fail though.

Reply:
@kir4h Because the `test` command without arguments exits with `1` exit status.

Reply:
Disregard my comment. I got the test right, but a combination of a typo in my credential process script and the fact that I missed somehow `Command` being a variadic function led me to wronly believe it would fail if no arguments provided.